conditions.md

```diff
@@ -9,11 +9,15 @@ Description of conditions https://github.com/kubernetes/community/blob/master/co
 ## Conditions in openstack-k8s-operators podified controlplane operators
 * At the beginning of the reconciliation loop, controller adds a `Ready` condition which reflects the overall state of the service with `Status=Unknown`
 * When a controller performs a task, before it starts a step condition gets initialized to `Status=Unknown` e.g. DBReady, DBSync, Endpoint, ServiceRegistered, …
+* Conditions should always be re-initialized at the beginning of each reconcile. This is to ensure that changes to the spec
+  will reset conditions accordingly.
 * At the end of the reconcile loop the overall `Ready` condition gets set to `Status=True/False`, depending if the sub task conditions are fulfilled or not.
-* The service `Ready` condition with `Status=True` reflects that the service is configured, up and accessible.
-* If the `Ready` condition is `Status=False` or `Unknown` the service is not OK to answer requests
-  - `Status=False` indicates an Error that needs user intervention to fix it
-  - `Status=Unknown` indicates that the operator can progress without user intervention. i.e the operator is waiting for a job to complete
+* The `Ready` condition with `Status=True` reflects that the service is configured, up and accessible and deployed with
```

Comment:
We would word this in a more generic way as not all of our CRDs describe a single service.
"
The Ready condition on our CRDs reflect that the expected state described by the Spec field of the CRD is the same as the actual state of the system, i.e. the operator finished reconciling the state of the system according to the Spec.
"

Reply:
works for me